### ISSUE TITLE
'NoneType' object has no attribute 'decode'

### ISSUE THREAD

**Issue by DannyMeister** [NotJoeMartinez/yt-fts#181]
I am not able to download videos from the channels I have tried.  Each gives the error message:

`'NoneType' object has no attribute 'decode'`

I see that in the past a similar issue was due to Youtube changes to the xml messing with page scraping.  Perhaps there has been another change interfering.

In case I've just been unlucky so far, an example of a broken channel is https://www.youtube.com/@CoachKarakal

**Comment by NotJoeMartinez** [comment]
What platform are you on (windows/mac/linux)? What version are you running (`yt-fts --version`)? Was does the full error look like? I was able to get a full download of the channel on MacOS and Linux.

<img width="860" alt="image" src="https://github.com/user-attachments/assets/9910c38d-70ad-4793-b4b6-2dc9f90f80c1">

**Comment by TeddyBear06** [comment]
I confirm too that I was able to get a full download of the channel on Windows 11.

![WindowsTerminal_nK1L7zENMk](https://github.com/user-attachments/assets/dc93fe62-0bb7-436f-bb50-4991f2888ce2)

A full dump of the error will help for sure :-)

**Comment by DannyMeister** [comment]
Well, it worked!  I rebuilt a new PC between today and when I encountered the issue, so I'm not going to be able to replicate.

I was on Windows 11 (still am) and using version 0.1.57.  I do know I was using the Edge browser and assumed it would be good to use my logged in user cookies, so the two differences between now and then are:
* A whole new PC!
* I was using --cookies-from-browser edge

Given that this worked first shot this time without cookies, I'm good to go.  Feel free to just close this issue, but I'll leave that to you instead of immediately closing, just in case you wanted to investigate the Windows/Edge browser combo.

Thank you!
